contexts/FinishedBooksContext.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { createContext, useState } from "react";
+
+export const FinishedBooks = createContext();
+
+function FinishedBooksProvider(props) {

```

Comment:
please convert to anonymous functions

```suggestion
const FinishedBooksProvider = (props) => {

```